README.md

```diff
@@ -1,6 +1,24 @@
 # Argo
 
-Argo is an extensible framework that allows rendering of remote content inside of Shopify's apps.
+Argo allows apps to integrate directly into the Shopify admin, mobile apps, checkout, or POS. It includes a set of consistent UI components, and development tools for authoring them.
```

Comment:
Since we're using the terminology _"surfaces"_ later, might as well introduce it as early as possible ...

```suggestion
Argo allows apps to integrate directly into Shopify surfaces such as Shopify admin, mobile apps, checkout, or POS. It includes a set of consistent UI components, and development tools for authoring them.
```

Also ... we don't support POS yet. Should we make it explicit?

Reply:
If we dont support it yet, we should remove it.